api/src/test/java/com/alibaba/nacos/api/naming/pojo/ServiceInfoTest.java

```diff
@@ -21,30 +21,29 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import org.junit.Before;
 import org.junit.Test;
-
 import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
 
 public class ServiceInfoTest {
-    
+
```

Comment:
Please use nacos code style to reformat. 

you can see relative information in source code `style/codeStyle.md`